src/Modules/ModParser.lua

```diff
@@ -2550,6 +2550,9 @@ local specialModList = {
 		mod("ShockBase", "BASE", data.nonDamagingAilment["Shock"].default, { type = "ActorCondition", actor = "enemy", var = "ChilledByYourHits" }),
 		mod("EnemyModifier", "LIST", { mod = flag("Condition:Shocked", { type = "Condition", var = "ChilledByYourHits" }) })
 	},
+	["enemies chilled by your hits lessen their damage dealt by half of chill effect"] = { 
+		mod("EnemyModifier", "LIST", { mod = mod("EnemyChillDamageReduction", "MAX", 1, { type = "PerStat", stat = "CurrentChill", from_enemy = true, div = 2 }, { type = "Condition", var = "ChilledByYourHits" })})
```

Comment:
```suggestion
		mod("EnemyModifier", "LIST", { mod = mod("EnemyChillDamageReduction", "MAX", 1, { type = "PerStat", stat = "CurrentChill", actor = "enemy", div = 2 }, { type = "Condition", var = "ChilledByYourHits" })})
```
I feel the only apply once stuff can be handled a bit better either by changing it to a flag and handling it entirely that way or by implementing a new globalLimit thing that will only apply a mod with a given globalThing once e.g. which would get rid of the need for your special tabulation code and couple be used in other similar situations in the future.